streamer/src/nonblocking/testing_utilities.rs

```diff
@@ -155,7 +190,7 @@ pub async fn make_client_endpoint(
     ));
     endpoint
         .connect(*addr, "localhost")
-        .expect("Failed in connecting")
+        .expect("Connecting should be configured correctly.")
         .await
-        .expect("Failed in waiting")
+        .expect("Connection should be able to be established.")
```

Comment:
*minor*

For the first assertion, I think, phrasing the expectations using the present
tense makes them even simpler.

For the second assertion, it is better to actually explain why do we think it
should not be failing, rather then just stating a rather obvious "the operation
succeeds" just in a slights different words:

```suggestion
    endpoint
        .connect(*addr, "localhost")
        .expect("Endpoint configuration is correct")
        .await
        .expect("Test server is already listening on the 'localhost'")
```

Reply:
Agree that you messages are a bit cleaner, but regarding "should": in the "Recommended Message Style" for expect it is recommended to use should.

Reply:
I guess, it should look like this then:

```rust
    endpoint
        .connect(*addr, "localhost")
        .expect("Endpoint configuration should be correct")
        .await
        .expect("Test server should be already listening on the 'localhost'")
```
https://doc.rust-lang.org/std/error/index.html#common-message-styles

I think present tense is a bit easier to read and reads fine in the context of the `expect()` call.
But, when phrased using "should", in case of an actual failure, the produced error message might look a bit more consistent.

Reply:
Ah, you already phrased it exactly like I was suggesting!